src/ggml-cuda/CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ if (CUDAToolkit_FOUND)
 
     set(CUDA_CXX_FLAGS "")
 
-    set(CUDA_FLAGS -use_fast_math)
+    set(CUDA_FLAGS -use_fast_math --threads=0 --split-compile=0)
```

Comment:
should we better create a new cmake option: eg CUDA_COMPILE_THREADS ?

Reply:
yea thats an easy fix. let me do that